lib/gitlab.rb

```diff
@@ -21,11 +21,18 @@ def self.client(options = {})
     Gitlab::Client.new(options)
   end
 
-  # Delegate to Gitlab::Client
-  def self.method_missing(method, *args, &block)
-    return super unless client.respond_to?(method)
-
-    client.send(method, *args, &block)
+  if RUBY_VERSION >= '3.0.0'
```

Comment:
I think you should use `Gem::Version` to compare the version strings. See doc [here](https://docs.ruby-lang.org/en/2.6.0/Gem/Version.html).

Reply:
You are right, that approach is a much better. Thanks.